static/src/javascripts/projects/common/modules/commercial/creatives/add-tracking-pixel.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'common/utils/template',
+    'text!common/views/commercial/tracking-pixel.html'
+], function (
+    template,
+    trackingPixelStr
+) {
+    var trackingPixelTpl;
```

Comment:
I'm not a big fan of abbreviated variable names.

If you want to make this more concise, I would just drop the 'trackingPixel' prefix (which can be inferred from the context) and merely call this a `template`.